chainercv/evaluations/eval_detection_voc.py

```diff
@@ -23,30 +23,33 @@ def eval_detection_voc(
     The code is based on the evaluation code used in PASCAL VOC Challenge.
 
     Args:
-        pred_bboxes (list of numpy.ndarray): A list of :math:`N` bounding
-            boxes. Its index corresponds to an index for the base dataset.
+        pred_bboxes (iterable of numpy.ndarray): An iterable of :math:`N`
```

Comment:
I didn't knew that. Thanks.
I confirmed this on the internet.
" an iterable is an object that you can get an iterator from."
from https://stackoverflow.com/questions/9884132/what-exactly-are-pythons-iterator-iterable-and-iteration-protocols